src/collections/_documentation/enriching-error-data/user-feedback-example/aspnetcore.md

```diff
@@ -2,7 +2,7 @@ With ASP.NET Core MVC, the `Error.cshtml` razor page:
 
 
 ```html
-<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.min.js" crossorigin="anonymous"></script>
+<script src="https://browser.sentry-cdn.com/{% sdk_version sentry.javascript.browser %}/bundle.min.js" integrity="sha384-{% sdk_cdn_checksum sentry.javascript.browser latest bundle.min.js %}" crossorigin="anonymous"></script>
```

Comment:
Could we perhaps move "sha384-" to the tag return value? It will be much easier to switch to other hash types in the future.